src/tests/ftest/object/object_integrity.py

```diff
@@ -135,10 +135,12 @@ def test_invalid_tx_commit_close(self):
             (1)DAOS-1346: Verify commit tx bad parameter behavior.
             (2)DAOS-1343: Verify tx_close bad parameter behavior.
             (3)DAOS-1342: Verify tx_close through daos_api.
+            (4)DAOS-1338: Add and verify tx_abort through daos_api.
+            (5)DAOS-1339: Verify tx_abort bad parameter behavior.
         Test Description:
             Write Avocado Test to verify commit tx and close tx
                           bad parameter behavior.
-        :avocado: tags=all,pr,small,negative_test,neg_tx_commit,medium,vm
+        :avocado: tags=all,object,full_regression,small,single_object
```

Comment:
unique test tag is missing.

Reply:
same as above...
Please see the fix in the new commit.